rmf_fleet_adapter/src/rmf_fleet_adapter/agv/FleetUpdateHandle.cpp

```diff
@@ -251,15 +254,25 @@ std::shared_ptr<rmf_task::Request> FleetUpdateHandle::Implementation::convert(
         const auto& p_value = (*p_it)["value"];
         if (p_value.is_number_integer())
         {
+          // If we have an integer greater than 0, we assign a high priority.
+          // Else the priority will default to low.
           if (p_value.is_number_integer() && p_value.get<uint64_t>() > 0)
+          {
             priority = rmf_task::BinaryPriorityScheme::make_high_priority();
+          }
         }
```

Comment:
I'll add an `else { invalid_schema = true; }` here to catch cases where the `"value"` isn't a number.